packages/app-trash-bin/src/Presentation/TrashBin/TrashBin.tsx

```diff
@@ -31,9 +31,10 @@ export interface TrashBinProps {
     restoreGateway: ITrashBinRestoreItemGateway<any>;
     itemMapper: ITrashBinItemMapper<any>;
     onClose: () => void;
-    onItemRestore: (item: TrashBinItemDTO) => Promise<void>;
+    onNavigateAfterRestoreItem: (item: TrashBinItemDTO) => Promise<void>;
```

Comment:
Why is this callback specific to "navigate", I mean, why not a more generic form, like: `onAfterRestoreItem`?